packages/eas-cli/src/update/configure.ts

```diff
@@ -149,7 +150,7 @@ async function ensureEASUpdatesIsConfiguredInExpoConfigAsync({
       return {
         projectChanged: true,
         // TODO(cedric): fix return type of `modifyConfigAsync` to avoid `null` for type === success repsonses
```

Comment:
yep, this `modifyConfigAsync` is probably a good thing to version next. We versioned `syncUpdatesConfigurationAsync` by creating a CLI for it in expo-updates.